template.tpl

```diff
@@ -414,7 +463,3 @@ scenarios: []
 
 
 ___NOTES___
-
-Created on 22.6.2021 08.53.59
```

Comment:
Is removing the timestamp the correct approach, or should we just update it?
I'm not 100% sure where (or if) it's needed.

Reply:
Obs: Resolved internally. The date is optional.

Reply:
Optional, so fine to remove